src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: lightgray;
+  padding: 1em;
+
+  &__title {
```

Comment:
The CSS class '.&__title' should follow BEM naming convention for better readability and maintainability. Consider renaming it to '.post-info__title'.